<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>165</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="DeviceComboBox">
      <item>
       <property name="text">
        <string>AD9850/51 with DG8SAQ USB i/f</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VNA MAX wiith Serial/USB i/f</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="ClockDoubleSpinBox">
      <property name="suffix">
       <string>MHz</string>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
      <property name="value">
       <double>125.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="ClockLabel">
      <property name="text">
       <string>DDS Clock</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="DeviceLabel">
      <property name="text">
       <string>Device</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="SerialPortLabel">
      <property name="text">
       <string>Serial Port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="SerialComboBox"/>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="MinFrequencyDoubleSpinBox">
      <property name="suffix">
       <string>MHz</string>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="MaxFrequencyDoubleSpinBox">
      <property name="suffix">
       <string>MHz</string>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="MinFrequencyLabel">
      <property name="text">
       <string>Min Frequency</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="MaxFrequencyLabel">
      <property name="text">
       <string>Max Frequency</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="MultiplierCheckBox">
      <property name="text">
       <string>Use multiplier</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
